psa_car_controller/web/view/api.py

```diff
@@ -181,6 +186,27 @@ def settings_section(section: str):
         APP.config.write_config()
     return json_response(config_section.json())
 
+@app.route('/get_vehicletrips')
+def get_trips():
+    try:
+        car = APP.myp.vehicles_list[0]
+        trips_by_vin = Trips.get_trips(Cars([car]))
+        trips = trips_by_vin[car.vin]
+        trips_as_dict = trips.get_trips_as_dict()
+        return FlaskResponse(json.dumps(trips_as_dict, default=str), mimetype='application/json')
```

Comment:
Could you use jsonify like this :
```suggestion
        return jsonify(trips_as_dict)
```

Reply:
Yeah that looks better indeed! I just updated :)